jujugui/static/gui/src/app/store/env/bakery.js

```diff
@@ -395,27 +415,35 @@ YUI.add('juju-env-bakery', function(Y) {
       }
       // Check that the condition actually holds.
       check(plain.Condition);
-      var rootKey = sjcl.codec.base64.toBits(plain.RootKey);
-      
+      var rootKey = [];
+      try {
+        var str = atob(plain.RootKey);
+        for (var i = 0; i < str.length; ++i) {
+          rootKey.push(str.charCodeAt(i));
+        }
+      } catch (ex) {
+        throw new Exception('Unable to parse RootKey from unsealed');
+      }
       return macaroon.newMacaroon(rootKey, caveatId, '');
     },
 
     /**
-      Default visit method which is to open a window to the
-      response.Info.VisitURL.
+     Default visit method which is to open a window to the
+     response.Info.VisitURL.
 
-      @method _defaultVisitMethod
-    */
+     @method _defaultVisitMethod
+     @return {undefined} Nothing.
+     */
     _defaultVisitMethod: function(response) {
       window.open(response.Info.VisitURL, 'Login');
     },
 
     /**
-      Get macaroon from local cookie.
+     Get macaroon from local cookie.
 
-      @method _getMacaroon
-      @return {String} Macaroons that was set in local cookie.
-    */
+     @method _getMacaroon
+     @return {String} Macaroons that was set in local cookie.
```

Comment:
s/Macaroons/Macaroon/

(isn't this assuming there can only be one macaroon in the local cookie store? is that correct?)